src/_assets/js/archive.js

```diff
@@ -1,7 +1,7 @@
 // TODO(chalin): package archive.js as its own top-level js so that it can be loaded independently
 
 // Number of releases to show by default (rest will be behind a "show all" link).
-var releasesToShow = 5;
+var releasesToShow = 99999;
```

Comment:
?

Reply:
I didn't want to remove the logic used to partition the entries (in case the approach used in this PR wasn't approved).